packages/legacy/babel/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@webex/babel-config-legacy",
+  "private": true,
+  "packageManager": "yarn@3.4.1",
+  "engines": {
+    "node": ">=18.0.0",
+    "npm": ">=8.0.0",
+    "yarn": ">=3.0.0"
+  },
+  "type": "commonjs",
+  "main": "./static/index.js",
+  "exports": {
+    ".": "./static/index.js",
```

Comment:
so this package will take care of babel config and version bump 
Will build legacy be a different one ?

Reply:
Yes, this babel config is a global config that is used by ESLint and out build workflows. This needs to be a separate package for this reason.

* `./legacy/build` => consumes `./legacy/babel`
* `./legacy/eslint` => consumes `./legacy/babel`